subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy

```diff
@@ -436,4 +437,74 @@ class ThirdPartyPluginsSmokeTest extends AbstractSmokeTest {
         )
     }
 
+    @Issue("https://plugins.gradle.org/plugin/com.google.protobuf")
+    def "protobuf plugin"() {
+        given:
+        buildFile << """
+            plugins {
+                id('java')
+                id("com.google.protobuf") version "${TestedVersions.protobuf}"
+            }
+
+            ${mavenCentralRepository()}
+
+            def protobufVersion = '3.11.1'
+            protobuf {
+                protoc {
+                    artifact = "com.google.protobuf:protoc:${'$'}protobufVersion"
```

Comment:
Use `\$` instead?

Reply:
Heh, I'm too used to Kotlin I guess :)
Anyway, after other changes this is now irrelevant.